examples/exch-retail.yaml

```diff
@@ -32,5 +32,5 @@
     resourceClassifiedAs: https://www.wikidata.org/wiki/Q4917 # US dollar
     resourceInventoriedAs: carol:e56fd654-7b94-4d96-8e60-de39e08329a7 # Carol's bank account
     resourceQuantity:
-      qudt:unit: unit:Number
-      qudt:numericValue: 5
+      om2:hasUnit: om2:one
```

Comment:
s/b http://www.ontology-of-units-of-measure.org/resource/om-2/UnitedStatesDollar ?